eng/build.yml

```diff
@@ -17,6 +17,8 @@ parameters:
   # Optional: Docker image to use
   dockerImage: ''
 
+  crossBuild: false
```

Comment:
Do we use this parameter?

Reply:
No its part of a separate change...  I'll fix it.